### PR TITLE
Version 1 

### PR DESCRIPTION
**Pull request by dansydoodles**

### CODE REVIEW

File: components/about.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import Image from "next/image";
+import circle from "../images/full-circle.png";
+import design from "../images/left-design.png";
+
+export const About = () => {
+  return (
+    <div id="about" className={"my-32 pb-20"}>
+      {/* <div
```
Comment (erick303) [inline components/about.tsx]: We can remove this commented out code

File: components/about.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              a more transparent and open financial market.
+            </p>
+            <p
+              className={
+                "text-text py-2 pt-4 font-roboto text-lg font-normal "
+              }
+            >
+              mZAR is a fully collaterized South African ZAR stablecoin, based
+              on the open fiat stablecoin framework of Mesh. mZAR is fully
+              reedemable for ZAR held in an account managed by Mesh.{" "}
```
Comment (erick303) [inline components/about.tsx]: Remove the {" "}

File: components/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          </ul>
+        </div>
+      </div>
+      <div>
+        <p
+          className={
+            " mb-2 px-10 font-poppins text-xs text-center text-[#171433] dark:text-[#707070] md:py-4 md:text-center lg:text-left xl:py-2 xl:px-8 "
+          }
+        >
+          Copyright 2021 Mesh,Inc. All rights reserved | MESH 87 00-844
```
Comment (erick303) [inline components/footer.tsx]: use {\`Copyright ${new Date().getFullYear()} Mesh,Inc. All rights reser.....\`} instead to have the date dynamically set to the curren year

File: components/hero.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
+import React, { MutableRefObject } from "react";
+import Image from "next/image";
+import mZAR from "../images/mZAR-coin.png";
+import Creative from "../images/mesh-creative.png";
+import { TrustedBySection } from "./trusted-by-section";
+
+export interface HeroProps {
+  // ref: MutableRefObject<HTMLDivElement | null>;
```
Comment (erick303) [inline components/hero.tsx]: we can delete the HeroProps since it's not used

File: components/hero.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+            {/* FEATURE BACKGROUND IMAGE */}
+            <div className="absolute bottom-0 -translate-x-14 translate-y-10 scale-[4] sm:scale-[3.3] sm:-translate-x-10 sm:translate-y-14 md:-translate-x-34 md:scale-[3] md:translate-y-28  lg:translate-x-10 lg:translate-y-10 lg:scale-[1.7] xl:translate-x-10 xl:translate-y-32  xl:scale-[1.5] 2xl:scale-[1.4] 2xl:translate-y-64 3xl:scale-[1.4]   3xl:translate-y-[27%] 3xl:translate-x-36 4xl:scale-[1.4]  4xl:translate-x-20 4xl:translate-y-[26%] 5xl:translate-x-28 5xl:translate-y-[28%] 5xl:scale-[1.3]">
+              <Image
+                src={Creative}
+                alt={"Mesh mZAR loop design"}
+                className={"rotate-[-5deg]"}
+                priority
+                quality={50}
+                // width={400}
```
Comment (erick303) [inline components/hero.tsx]: remove the comments

File: components/nav-bar.tsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
+//import type { NextPage } from "next";
```
Comment (erick303) [inline components/nav-bar.tsx]: remove comment

File: components/nav-bar.tsx [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+import Image from "next/image";
+import mZARLogo from "../images/mzar-logo.svg";
+import React, { useRef, useState } from "react";
+import Link from "next/link";
+
+import { Menu } from "@headlessui/react";
+
+import cn from "classnames";
+import useIntersectionObserver from "../hooks/useIntersectionObserver";
```
Comment (erick303) [inline components/nav-bar.tsx]: this hook does not seem to be used here

**Review by erick303 — CHANGES_REQUESTED** [review]
Can we make the public/logo.png image have a transparent background?

File: components/about.tsx (hunk 1 above)
Comment (dansydoodles) [reply components/about.tsx]: removed

File: components/about.tsx (hunk 2 above)
Comment (dansydoodles) [reply components/about.tsx]: removed

File: components/footer.tsx (hunk 3 above)
Comment (dansydoodles) [reply components/footer.tsx]: Replaced

File: components/hero.tsx (hunk 4 above)
Comment (dansydoodles) [reply components/hero.tsx]: deleted

File: components/hero.tsx (hunk 5 above)
Comment (dansydoodles) [reply components/hero.tsx]: removed

File: components/nav-bar.tsx (hunk 6 above)
Comment (dansydoodles) [reply components/nav-bar.tsx]: removed

File: components/nav-bar.tsx (hunk 7 above)
Comment (dansydoodles) [reply components/nav-bar.tsx]: removed

**Review by erick303 — APPROVED** [review]

**Pull request merged**